src/components/HeroSection/Hero.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "../../globals.css";
```

Comment:
- no need to import react as mentioned in the discord chat
- no need to import any css